Client/Frontend/Browser/Playlist/Controllers/PlaylistListViewController.swift

```diff
@@ -343,6 +344,10 @@ class PlaylistListViewController: UIViewController {
             PlaylistManager.shared.fetchedObjects[safe: $0.row]
         })
         
+        if selectedItems.contains(where: { $0.pageSrc == PlaylistCarplayManager.shared.currentPlaylistItem?.pageSrc }) {
```

Comment:
Is it added because a playlist item should be stopped playing in car entirely when it is moved  to another folder in phone ?

Reply:
Yes that is correct. If the current item is moved, it needs to stop playing. It doesn't make sense to keep playing when we're moving the current item to a new folder. Otherwise the person will be able to pause and unpause an item that isn't in the current folder.